### DISCUSSION TITLE
Not able to create feature branch sanjay

### DISCUSSION THREAD

**Discussion by harshita2303** [sanjay-kv/Open-source-Practice#1192] [Q&A] [answered]
@sanjay-kv  why i am getting 403 error while creating a feature branch sanjay

**Comment by sanjay-kv** [comment]
scrrenshot please

**Reply by harshita2303** [reply]
![IMG-20241022-WA0002](https://github.com/user-attachments/assets/d96fe1ed-89b2-4cde-8c93-066af1643e66)

**Reply by sanjay-kv** [reply]
you have to fork

**Comment by codeftsaikat** [comment, marked as answer]
You're getting a 403 error likely because

Insufficient permissions – Check if you have write access to the repo.
Pushing to the original repo instead of your fork – Make sure you’re on your forked repo.
Authentication issue – Update your GitHub token or SSH key.

**Comment by hirushavinushka99** [comment]
A 403 error usually occurs due to:  

Insufficient Permissions: Ensure you have write access to the repository.  
Pushing to the Wrong Repo: Verify that you're working on your forked repository, not the original one.  
Authentication Problems: Update your GitHub token or SSH key to resolve access issues.
